src/bin/inx-chronicle/api/stardust/poi/error.rs

```diff
@@ -4,12 +4,12 @@
 use thiserror::Error;
 
 #[derive(Error, Debug)]
-#[allow(missing_docs)]
+#[allow(missing_docs, clippy::enum_variant_names)]
```

Comment:
Just name them better 😆

Reply:
Any suggestions? Because I like them how they are named. It's also a stupid clippy lint. What is it helping with exactly?

Reply:
Just remove the prefix `Invalid`.

Reply:
I know that this is what I am supposed to do, but what if I don't agree? It would be inconsistent with how we name other error variants: for instance `InxError`. Should we remove `Invalid` and `Missing` there, too? This clippy warning should belong to the pedantic group of suggestions as it doesn't help with improving the code in any significant way.

I am fixing this in another way.